pyvista/utilities/geometric_objects.py

```diff
@@ -968,6 +970,13 @@ def Pyramid(points):
     >>> pyramid = pyvista.Pyramid([pointa, pointb, pointc, pointd, pointe])
     >>> pyramid.plot(show_edges=True, line_width=5)
     """
+    if points is None:
+        points = [[1.0, 1.0, 0.0],
+                  [-1.0, 1.0, 0.0],
+                  [-1.0, -1.0, 0.0],
+                  [1.0, -1.0, 0.0],
+                  [0.0, 0.0, np.sqrt(2.0**2-2.0*np.sqrt(2.0)/2.0)]]
```

Comment:
1. This last number is approximately 1.61 rather than 1.67, so technically the docstring fibs about the default being the example's pyramid.
2. Without an explanation, does the formula have to be this complicated? What I'm seeing is `np.sqrt(4 - np.sqrt(2))`.

Reply:
```suggestion
                  [0.0, 0.0, (4 - 2**0.5)**0.5]]
```
Not much of a reason for `np.sqrt` since it's a single value.

Reply:
> This last number is approximately 1.61 rather than 1.67, so technically the docstring fibs about the default being the example's pyramid.

Will fix in a separate commit.